app/models/container_deployment_custom_setting.rb

```diff
@@ -0,0 +1,3 @@
+class ContainerDeploymentCustomSetting < ApplicationRecord
+  belongs_to :container_deployment
+end
```

Comment:
@Fryguy we thought it would be better to keep it separate, if you are fine with letting us use the new table ill remove it :)